Cargo.toml

```diff
@@ -18,14 +18,14 @@ build = "build.rs"
 [dependencies]
 # Substrate crates.
 scale = { package = "parity-scale-codec", version = "3", default-features = false, optional = true }
-scale-info = { version = "2.1.1", default-features = false, features = ["derive"], optional = true }
+scale-info = { version = "2.3.1", default-features = false, features = ["derive"], optional = true }
```

Comment:
Lets keep substrate related crate the same for now, since it is also tied to the runtime and subxt crates.